textproc/aspell-dict-da/Portfile

```diff
@@ -16,10 +16,11 @@ supported_archs     noarch
 
 master_sites	    gnu:aspell/dict/${langcode}
 
-distname	        aspell5-${langcode}-${version}
-checksums           md5     d14c03dca23b572606279d7317b022d0 \
-                    sha1    8cc990c195707e2b4db4ed0969aa10117a3b6bb2 \
-                    rmd160  e9278ec5665d54066ff1808176e08af280501b9c
+distname	        aspell6-${langcode}-${version}
+checksums           sha1    4c5a3f5bc8ebf5453338f4279de5f5474888edd4 \
```

Comment:
You seem to fairly consistently be keeping the `sha1` checksum. Is it provided by upstream for verification? If not, then please remove it here and in the other Portfiles you updated.

Reply:
Since the original `Portfile` contains `md5` and `sha1`, running `port -ds checksum` prints these values along with the others (when values don't match). I just copy-paste this output into the `Portfile` since it is in the correct format already. I knew to delete `md5` but not the `sha1`. The PR is now updated without the `sha1` values.